### DISCUSSION TITLE
Duda practica pasteleria

### DISCUSSION THREAD

**Discussion by jjmillar** [Launch-X-Latam/MisionFrontEnd#311] [Q&A] [answered]
Buenas compas explorers!! Consulta, en que live sale la practica de la pasteleria? Consulto pk terminé de ver todos los lives de CSS y no vi la practica. Quizas se me paso o no puse la atencion necesaria, pero me podrían decir donde sale la practica de la pasteleria? O es la misma de la taquería sumandole css? Gracias colegas explorers!

**Comment by Amhedriel** [comment, marked as answer]
Buenas colega, la práctica de "pastelería" como tal no la toco exactamente como se hizo con UI UX, en el caso de *pastelería* lo trato más directamente, pero lo hizo con otro ejemplo, en este caso recordaras "Las taquerías del jefe" porque con tantos participantes ya es una franquicia jeje, pero bueno, en ese caso nos dijo que puedes ponerle, taqueria, pastelería, dulcería, etc. lo importante es hacer la práctica con los requerimientos que nos solicita el cliente, saludos.

**Comment by AlbertoSF99** [comment]
Aquí mismo en Github te vas al apartado de "MisionFrontEnd" y te vas a la carpeta "02 - HTML" para después buscar la carpeta "practicas". Ahí encontrarás los requisitos que debes cumplir para hacer tu práctica.

**Comment by zaazilcb** [comment]
Hola compañero, como mencionan en los otros comentarios, la practica de pastelería es del tema de HTML y lo puedes encontrar aqui en github en el apartado de prácticas en la sección de html donde se especifica los requerimientos del cliente y lo que se tiene que entregar asi como darle un plus a tu entrega si te queda corta la práctica. Tambien en la ultima clase de esta misma realizamos la práctica de "la taqueria el jefe" la cual es una práctica muy similar a la pasteleria solo que con otros requerimiento. Como quizá hayas notado algunos compañeras dejaron muy bonita su página de pasteleria con diseño y demás in embargo lo que te piden en la práctica es solo la maquetación, ya la parte de diseño es un extra.
